graylog2-web-interface/src/components/navigation/HeaderBadge.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import AppConfig from 'util/AppConfig';
+import { Badge } from 'components/graylog';
+
+type Props = {
+  smallScreen: boolean,
+};
+
+const HeaderBadge = ({ smallScreen }: Props) => {
```

Comment:
I think this should indicate that it's not a generic header badge, but the `DevelopmentHeaderBadge` or something similar.